assets/js/romo/dropdown.js

```diff
@@ -88,90 +87,80 @@ RomoDropdown.prototype.doInitPopup = function() {
-  });
+  Romo.setStyle(this.contentElem, 'min-height', Romo.data(this.elem, 'romo-dropdown-min-height'));
+  Romo.setStyle(this.contentElem, 'height',     Romo.data(this.elem, 'romo-dropdown-height'));
+  Romo.setStyle(this.contentElem, 'overflow-x', Romo.data(this.elem, 'romo-dropdown-overflow-x') || 'auto');
+  Romo.setStyle(this.contentElem, 'overflow-y', Romo.data(this.elem, 'romo-dropdown-overflow-y') || 'auto');
 
-  if (this.elem.data('romo-dropdown-max-height') === undefined) {
-    this.elem.attr('data-romo-dropdown-max-height', 'detect');
+  if (Romo.data(this.elem, 'romo-dropdown-max-height') === undefined) {
+    Romo.setData(this.elem, 'romo-dropdown-max-height', 'detect');
   }
-  if (this.elem.data('romo-dropdown-max-height') !== 'detect') {
-    this.contentElem.css({
-      'max-height': this.elem.data('romo-dropdown-max-height')
-    });
+  if (Romo.data(this.elem, 'romo-dropdown-max-height') !== 'detect') {
+    Romo.setStyle(this.contentElem, 'max-height', Romo.data(this.elem, 'romo-dropdown-max-height'));
   }
 
-  if (this.elem.data('romo-dropdown-width') === 'elem') {
-    this.popupElem.css({
-      'width': this.elem.css('width')
-    });
+  if (Romo.data(this.elem, 'romo-dropdown-width') === 'elem') {
+    Romo.setStyle(this.popupElem, 'width', Romo.css(this.elem, 'width'));
   } else {
-    this.contentElem.css({
-      'min-width':  this.elem.data('romo-dropdown-min-width'),
-      'max-width':  this.elem.data('romo-dropdown-max-width'),
-      'width':      this.elem.data('romo-dropdown-width')
-    });
+    Romo.setStyle(this.contentElem, 'min-width', Romo.data(this.elem, 'romo-dropdown-min-width'));
+    Romo.setStyle(this.contentElem, 'max-width', Romo.data(this.elem, 'romo-dropdown-max-width'));
+    Romo.setStyle(this.contentElem, 'width',     Romo.data(this.elem, 'romo-dropdown-width'));
   }
 }
 
 RomoDropdown.prototype.doResetBody = function() {
-  this.contentElem.css({
-    'min-width':  '',
-    'max-width':  '',
-    'width':      '',
-    'min-height': '',
-    'max-height': '',
-    'height':     '',
-    'overflow-x': '',
-    'overflow-y': ''
-  });
+  Romo.setStyle(this.contentElem, 'min-width',  '');
+  Romo.setStyle(this.contentElem, 'max-width',  '');
+  Romo.setStyle(this.contentElem, 'width',      '');
+  Romo.setStyle(this.contentElem, 'min-height', '');
+  Romo.setStyle(this.contentElem, 'max-height', '');
+  Romo.setStyle(this.contentElem, 'height',     '');
+  Romo.setStyle(this.contentElem, 'overflow-x', '');
+  Romo.setStyle(this.contentElem, 'overflow-y', '');
 }
 
 RomoDropdown.prototype.doLoadBodyStart = function() {
-  this.bodyElem.html('');
+  this.bodyElem.innerHTML = '';
   this.doInitBody();
   this.doPlacePopupElem();
-  this.elem.trigger('dropdown:loadBodyStart', [this]);
+  Romo.trigger(this.elem, 'romoDropdown:loadBodyStart', [this]);
 }
 
 RomoDropdown.prototype.doLoadBodySuccess = function(data) {
   Romo.initHtml(this.bodyElem, data);
```

Comment:
@jcredding this needs updating.  This method no longer exists.  I think it should be `initUpdate` now.